README.md

```diff
@@ -118,64 +119,64 @@ To build with support for the processor's internal instruction clock, use:
 Verbose build
 -------------
 
-PlaceHolderKV will build with a user-friendly colorized output by default.
+Valkey will build with a user-friendly colorized output by default.
 If you want to see a more verbose output, use the following:
 
     % make V=1
 
-Running PlaceHolderKV
+Running Valkey
 -------------
 
-To run PlaceHolderKV with the default configuration, just type:
+To run valkey with the default configuration, just type:
 
     % cd src
-    % ./placeholderkv-server
+    % ./valkey-server
 
-If you want to provide your placeholderkv.conf, you have to run it using an additional
+If you want to provide your valkey.conf, you have to run it using an additional
 parameter (the path of the configuration file):
 
     % cd src
-    % ./placeholderkv-server /path/to/placeholderkv.conf
+    % ./valkey-server /path/to/valkey.conf
 
-It is possible to alter the PlaceHolderKV configuration by passing parameters directly
+It is possible to alter the valkey configuration by passing parameters directly
 as options using the command line. Examples:
 
-    % ./placeholderkv-server --port 9999 --replicaof 127.0.0.1 6379
-    % ./placeholderkv-server /etc/placeholderkv/6379.conf --loglevel debug
+    % ./valkey-server --port 9999 --replicaof 127.0.0.1 6379
+    % ./valkey-server /etc/valkey/6379.conf --loglevel debug
 
-All the options in placeholderkv.conf are also supported as options using the command
+All the options in valkey.conf are also supported as options using the command
 line, with exactly the same name.
 
-Running PlaceHolderKV with TLS:
+Running valkey with TLS:
 ------------------
 
 Please consult the [TLS.md](TLS.md) file for more information on
-how to use PlaceHolderKV with TLS.
+how to use valkey with TLS.
 
-Playing with PlaceHolderKV
+Playing with valkey
```

Comment:
General comment - Unsure the choice of upper case/lower case first letter in this PR. I see Valkey and valkey used interchangeably. However, I think for the binary references we should use valkey.

Reply:
Ok, I tried to be more consistent. Binaries are all lower case and naming is capitalized.

Reply:
I think this needs to be "Valkey"